pyproject.toml

```diff
@@ -54,16 +54,13 @@ exclude = '''
 
 [tool.pytest.ini_options]
 minversion = "6.0"
-addopts = "-ra -q --cov-report term-missing --cov=beanie"
+addopts = "--cov-report term-missing --cov=beanie --cov-branch --cov-fail-under=85"
 testpaths = [
     "tests",
-    "integration",
 ]
 filterwarnings = [
     "error",
-    "ignore::UserWarning",
```

Comment:
Without this some tests are red.

```>               warnings.warn(pytest.PytestUnraisableExceptionWarning(msg))
E               pytest.PytestUnraisableExceptionWarning: Exception ignored in: <socket.socket [closed] fd=-1, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6>
E               
E               Traceback (most recent call last):
E                 File "/home/runner/.cache/pypoetry/virtualenvs/beanie-EwvjBtbN-py3.9/lib/python3.9/site-packages/aiohttp/test_utils.py", line 536, in teardown_test_loop
E                   gc.collect()
E               ResourceWarning: unclosed <socket.socket fd=54, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('127.0.0.1', 60662), raddr=('127.0.0.1', 27017)>
```
Somewhere socket was not closed by the end of the test and it raised a warning. As user warnings are not ignored here, it marked this test as red.